component_sdk/python/tests/launcher/test_launcher.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import mock
+import unittest
+import math
+import os
+
+from kfp_component.launcher import launch
+
+TEST_PY_FILE = os.path.join(os.path.dirname(__file__), 'echo.py')
+
+class LauncherTest(unittest.TestCase):
+
+    def test_launch_module_succeed(self):
+        self.assertEqual(math.pi, launch('math', 'pi'))
+
+    def test_launch_py_file_succeed(self):
+        self.assertEqual('hello', 
+            launch(TEST_PY_FILE, ['echo', 'hello']))
```

Comment:
Could we have a unit test that covers launching a user-defined module since you already have launching a user-defined file, launch an installed module.

Reply:
I currently don't support user defined module without install in the system. I think python file should be able to cover that as well. User just need to point to the entrypoint python file in the module folder.

Reply:
I guess python file is sufficient now. I thought you supposed user defined module already and that was why I was asking for the unit tests.